src/html/hardware.html

```diff
@@ -174,6 +174,7 @@ <h1><b>ExpressLRS</b></h1>
 					</td><td>How the power level is set</td></tr>
 					<tr><td></td><td>Power Value(s)</td><td><input size='40' id='power_values' name='power_values' type='text' class='array'/></td><td>Comma-separated list of values that set the power output (if using a DAC these are the DAC values)</td></tr>
 					<tr><td></td><td>Secondary Power Value(s)</td><td><input size='40' id='power_values2' name='power_values2' type='text' class='array'/></td><td>Comma-separated list of values that set the power output (if using a DAC then these set the Semtech power output)</td></tr>
+					<tr><td></td><td>Dual Power Value(s)</td><td><input size='40' id='power_values_dual' name='power_values_dual' type='text' class='array'/></td><td>Comma-separated list of values that set the higher frequency power output of a dual band Tx/Rx</td></tr>
```

Comment:
I guess we should also have `power_values_dual2` for DAC output values for the high frequency band!?

Reply:
Yep.  But it was a bridge too far for me when doing this.

Reply:
I thought that might be the case, but wanted to capture the thought 😄

Reply:
Absolutely!  There are probably a few other things I haven't done yet,.. or completely forgotten about.